### ISSUE TITLE
Offset not found at 5% after webview shows rebel case ad

### ISSUE THREAD

**Issue by Iam2fast4u2catch** [pwn20wndstuff/Undecimus#2061]
**Describe the bug**
A clear and concise description of what the bug is and what you expected to happen.

Hi World.
i Can confirm same offset not found Error on Iphone 5S ios 11.2.5 after rebel ad pops up in webview that brakes the jb process.
i close popup it says fail... at 5% in 5.0.0 AND 5.0.1 AND the new 5.2.0

the 4.3.1 works flawlessly!
THX..

Greets from CH

![135B683F-0FEA-4302-BC57-888B98B18CB7](https://user-images.githubusercontent.com/40250023/84491464-e2542200-aca4-11ea-814b-34a67b3f1c4e.jpeg)

**To Reproduce**

**Screenshots**

**Device (please complete the following information):**
 - iOS Version: 11.2.5
 - iOS Device: 5S
 - unc0ver Version: 5.0.0 / 5.0.1 / 5.2.0

**Place an "x" between the brackets if true:**
 - [X] this is a bug others will be able to reproduce
 - [X] this issue is present with all tweaks uninstalled(except for default packages) or disabled
 - [X] this issue is present after a rootfs restore
 - [X] this issue is present on the latest version of unc0ver

**Logs**
If applicable, add logs or error messages here.

**Comment by JTCGaming** [comment]
I got this same problem too. I'm on iPhone 6 ios 11.4.1. I think they dropped support for us iphone 6 and below users... This is ridiculous. One would think they could have at least kept the older exploits for us older iphone users?

**Comment by Hamlock-maneuverr** [comment]
Try using an older version of unc0ver.  Not sure why you stayed on ios 11, but whatever. try version 3.8.0 for starters.

**Comment by Iam2fast4u2catch** [comment]
> Try using an older version of unc0ver. Not sure why you stayed on ios 11, but whatever. try version 3.8.0 for starters.

Hi.
As i said 4.3.1 works for me.
if i go to update ios of my iphone 5s i would get to 12.4.7 thats the highest 5s supported but 12.4.7 is NOT supported by any version of Undecimus.. 🤔

**Comment by JTCGaming** [comment]
I never updated to ios 12 because i waited on ios 11 for the jailbreak and when it came out for my version, I felt like there was no point on breaking it if it now works after waiting so long... Not to mention I only have the 16 GB model, so not a lot of room to work with.

Now how do I revert back to the older version? I looked all over the internet and found nothing that was helpful and nothing that wouldn't set me back $100 :/

**Comment by Iam2fast4u2catch** [comment]
revert back
to last working 4.3.1 undecimus ? 🤔
if you have a active working jailbreak you can look for the self
signing cydia app called reprovision and download the unsigned 4.3.1 ipa right from here... 

![FC29D993-85E9-4FA8-9099-E5F7671779B7](https://user-images.githubusercontent.com/40250023/84603116-31a87700-ae8c-11ea-945a-b25e96376921.jpeg)

**Comment by JTCGaming** [comment]
Unfortunately my jailbreak is not active because my phone had died. The problem is activating the jailbreak again to allow me to do that method.

**Comment by pbgbottle** [comment]
Yep all my devices on iOS 12.4 have the same issue. I have had to use AltStore and download v4.3.1 and sideload it that way and jailbreak. The v5 unc0ver doesn’t work on any of my ios12 devices. iPad mini 2 ,iPhone 5s ,iPhone 6plus. ,

**Comment by JTCGaming** [comment]
So over past couple days I figured out how to revert myself back to 4.3.1 from a unkailbroken state. Unfortunately it screwed up all my tweaks so I had to manually remove them and reinstall them again :,( but was worth it. Basically I used 3UTools to install a jailbreak app into my phone, which for some reason it installed Electra. I rejailbroke using Electra which reinstalled Cydia over the already installed Cydia and its tweaks. Installed ReProvision and used that to sideload unc0ver 4.3.1 onto my device. From there I ssh into my iPhone and threw a executable into my phone and ran it which removed Cydia and all the shit Electra puts on ur phone and rebooted. I dont exactly remember what the tool was but if anyone’s needs it, I guess just let me know. Anyway I think from there you would know what to do. :)

**Comment by pbgbottle** [comment]
I just used altstore and downloaded the older version of unc0ver and sideloaded it using the  altstore

**Comment by JTCGaming** [comment]
@pbgbottle I tried to do that but altstore is not supported on ios 11.

**Comment by Hamlock-maneuverr** [comment]
Now you understand why I was curious why you stayed on 11. IOS  app version compatibility is going to become an issue the lower your version is.  You want to be on the highest IOS possible for your device.

**Comment by JTCGaming** [comment]
@Hamlock-maneuverr yes you are right. Unfortunately I missed my window of opportunity to upgrade and now if I do, I'll be waiting for a new exploit to come out for IOS 12.4.7. I guess you can say I walked into this one for not upgrading when I should have lol, but at least I solved my own convoluted problem. And hopefully I could help someone else solve a similar problem.

**Comment by Iam2fast4u2catch** [comment]
.
   .
5.2.1 working as ad is show at end of JB process 😀
   .
   .

**Comment by Iam2fast4u2catch** [comment]
Case closed

**Comment by Iam2fast4u2catch** [comment]
.
